content/30.methods.md

```diff
@@ -26,79 +26,44 @@ for hyphenated names, all components were included.
 These methods extract a sequence of textual affiliations for each author.
-While ideally each affiliation record would refer to one and only one research organization, sometimes journals deposit multiple affiliations in a single structured affiliation.
-For example, we extracted the following composite affiliation for all authors of [PMC4147893](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4147893/):
-
-> 'Multimodal Computing and Interaction', Saarland University & Department for Computational Biology and Applied Computing, Max Planck Institute for Informatics, Saarbrücken, 66123 Saarland, Germany, Ray and Stephanie Lane Center for Computational Biology, Carnegie Mellon University, Pittsburgh, 15206 PA, USA, Department of Mathematics and Computer Science, Freie Universität Berlin, 14195 Berlin, Germany, Université Pierre et Marie Curie, UMR7238, CNRS-UPMC, Paris, France and CNRS, UMR7238, Laboratory of Computational and Quantitative Biology, Paris, France.
 
-We designed a method for extracting countries from affiliations that accommodated multiple countries.
 We relied on two Python utilities to extract countries from text: [geotext](https://github.com/elyase/geotext) and [`geopy.geocoders.Nominatim`](https://geopy.readthedocs.io/en/stable/#nominatim).
 The first, geotext, used regular expressions to find mentions of places from the [GeoNames database](http://www.geonames.org/).
-In the above text, geotext detected four mentions of places in Germany: Saarland, Saarbrücken, Saarland, Germany.
-Anytime geotext identified 2 or more mentions of a country, we labeled the affiliation as including that country.
-
-`geopy.geocoders.Nominatim` converts names / addresses to geographic coordinates using the OpenStreetMap's [Nomatim](https://nominatim.org/) service.
-We split textual affiliations by punctuation and found the first segment, in reverse order, that returned any Nomatim search results.
-For the above affiliation, the search order was "France", "Paris", "Laboratory of Computational and Quantitative Biology", etcetera.
-Since searching "France" returns a match by Nomatim, the following queries would not be made.
-When a match was found, we extracted the country containing the location.
-This approach returns a single country for an affiliation when successful.
-When labeling affiliations with countries, we only used these values when geotext did not return results or had ambiguity amongst countries without multiple matches.
-For more details on this approach, consult the accompanying [notebook](https://github.com/greenelab/iscb-diversity/blob/5213ba3451520af3967f74d8f58553dade0a826c/07.affiliations-to-countries.ipynb) and [label dataset](https://github.com/greenelab/iscb-diversity/blob/5213ba3451520af3967f74d8f58553dade0a826c/data/affiliations/geocode.jsonl).
+To avoid mislabeling, we only mapped the affiliation to a country if geotext identified 2 or more mentions of that country.
```

Comment:
get_consensus_countries still assigns countries when geotext does not have any countries with 2 or more mentions. At that point it involves geocode

Reply:
Perhaps this is already clear because this is part of the geotext paragraph

Reply:
Yes. This is the reason why I separated out the geocode section a couple paragraphs below.

> When geotext did not return results, we use the geopy approach, which returns a single country for an affiliation when successful.